src/constants.ts

```diff
@@ -4,3 +4,6 @@
  */
 
 export const MAX_BUNDLE_ENTRIES = 25;
+// Lambda payload limit is 6MB, assuming an average request of 4KB,
+// we have 6MB / 4Kb = 1500. Dividing by half to allow for contingencies, we have the following limit:
+export const MAX_BATCH_ENTRIES = 750;
```

Comment:
Can we make this a value that can be passed in from deployment package with a default of 750? Customers might want to customize this number.

Reply:
Done! I added it to the changes in the deployment package as well.